source/github-helpers/search-query.ts

```diff
@@ -66,6 +66,11 @@ export default class SearchQuery {
 			return '';
 		}
 
+		// Parse label links #5176
+		if (this.link.pathname.includes('/labels/')) {
+			return `is:open label:"${decodeURIComponent(/\/labels\/(.+)$/.exec(this.link.pathname)![1])}"`;
```

Comment:
There's a lot going on here, let's create at least one variable. Also you can use [^/]+ instead of .+, unless slashes are not escaped by GitHub.